js/views/file_input_view.js

```diff
@@ -301,6 +301,16 @@
         return;
       }
 
+      // Save temporary copy of file to disk.
```

Comment:
This PR doesn't properly handle all the message composition/send scenarios. We should definitely delete temp files wherever we can before the full directory delete on shutdown:
1. When the user removes staged attachments - see `onCloseAttachment()` and `onClose()` in this file
2. When the user sends the message - see `clearAttachments` in this file